eclipse-scout-core/src/tile/TileGrid.ts

```diff
@@ -463,12 +463,18 @@ export class TileGrid extends Widget implements TileGridModel {
     this.comparator = comparator;
   }
 
-  sort() {
+  protected _sortAfterInit() {
+    if (!this.sort()) {
+      this.updateFilteredElements();
```

Comment:
Is it necessary to call updateFilteredElements here? updateFilteredElements does only filter if filteredElementsDirty is true anyway -> Can we call it in init right after _sortAfterInit?